tests/test_data/test_datasets/test_sr_dataset.py

```diff
@@ -1,26 +1,27 @@
-import os.path as osp
 from pathlib import Path
 from unittest.mock import patch
 
 import numpy as np
 import pytest
-from torch.utils.data import Dataset
 
 # yapf: disable
-from mmedit.datasets import (AdobeComp1kDataset, BaseGenerationDataset,
-                             BaseSRDataset, GenerationPairedDataset,
-                             GenerationUnpairedDataset, RepeatDataset,
-                             SRAnnotationDataset, SRFacialLandmarkDataset,
-                             SRFolderDataset, SRFolderGTDataset,
-                             SRFolderMultipleGTDataset, SRFolderRefDataset,
-                             SRFolderVideoDataset, SRLmdbDataset,
-                             SRREDSDataset, SRREDSMultipleGTDataset,
-                             SRTestMultipleGTDataset, SRVid4Dataset,
-                             SRVimeo90KDataset, SRVimeo90KMultipleGTDataset)
+from mmedit.datasets import (BaseSRDataset, SRAnnotationDataset,
+                             SRFacialLandmarkDataset, SRFolderDataset,
+                             SRFolderGTDataset, SRFolderMultipleGTDataset,
+                             SRFolderRefDataset, SRFolderVideoDataset,
+                             SRLmdbDataset, SRREDSDataset,
+                             SRREDSMultipleGTDataset, SRTestMultipleGTDataset,
+                             SRVid4Dataset, SRVimeo90KDataset,
+                             SRVimeo90KMultipleGTDataset)
 
 # yapf: enable
 
 
+def check_keys_contain(result_keys, target_keys):
```

Comment:
This helper function is defined many times. Please

* Make a pull request to mmcv, see examples here https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/testing.py
* Adding it to `mmedit/utils/test_utils.py` and use it for now since mmcv is slow in response.